### PR TITLE
[TENJIN-14518] Update Flutter SDK with latest dependencies and native SDKs

### PR DESCRIPTION
**Pull request by giraldogdiego**
**JIRA**:

- https://adromance.atlassian.net/browse/TENJIN-14518

**Proposed Changes:**

- Update plugin with latest Tenjin SDKs
- Add missing methods
- Update Flutter, Android and iOS dependencies
- Use Maven and Cocoapods instead of binaries

**Steps:**

- [ ] Reviewed
- [ ] Deployed to Staging
- [ ] QA'd
- [ ] Deployed to Production

### CODE REVIEW

File: android/src/main/kotlin/com/tenjin_sdk/TenjinSdkPlugin.kt [hunk 1]
```diff
@@ -48,6 +49,9 @@ class TenjinSdkPlugin: FlutterPlugin, MethodCallHandler {
       "transaction" -> {
         transaction(call, result)
       }
+      "transactionWithReceipt" -> {
+        transaction(call, result)
```
Comment (nafeu) [inline android/src/main/kotlin/com/tenjin_sdk/TenjinSdkPlugin.kt]: As pointed out by Colin, this should not be transaction(...)

File: README.md [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 
-**[Tenjin Docs](https://docs.tenjin.com/en/)**
+## Plugin Integration
 
 1. Add the dependency to the `pubspec.yaml` file in your project:
 
 ```yaml
 dependencies:
-  tenjin_Sdk: any
+  tenjin_plugin: any
```
Comment (nafeu) [inline README.md]: Maybe use a fixed version or recommend a version alongside `any` being a possible option

File: .idea/libraries/Flutter_Plugins.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<component name="libraryTable">
+  <library name="Flutter Plugins" type="FlutterPluginsLibraryType">
+    <CLASSES>
+      <root url="file://$PROJECT_DIR$" />
+    </CLASSES>
+    <JAVADOC />
+    <SOURCES />
+  </library>
+</component>
```
Comment (caleon) [inline .idea/libraries/Flutter_Plugins.xml:9]: need a newline

**Review by nafeu — CHANGES_REQUESTED** [review]
Some minor changes

**Review by caleon — APPROVED** [review]

**Pull request merged**